Rakefile

```diff
@@ -30,12 +30,10 @@ task :docstats, [:path] do |_, args|
   sh('yard', 'stats', '--list-undoc', path)
 end
 
-Rake::ExtensionTask.new('drawText')
-
 GEM_NAME = 'fastlane-plugin-wpmreleasetoolkit'.freeze
 VERSION_FILE = File.join('lib', 'fastlane', 'plugin', 'wpmreleasetoolkit', 'version.rb')
 
-desc 'Try to build and install the gem to ensure it can be installed properly (with the native extension and all)'
+desc 'Try to build and install the gem to ensure it can be installed properly'
```

Comment:
Took the liberty to push that change myself here, as this rake task was added mostly to ensure that the drawText native extensions was able to build.

We might even consider removing this task? Or at least removing the `check_build` step that is made during `new_release` task, as now that the CI can check that the gem builds for us, no need to do the same check locally before creating the PR… even less so now that `drawText` won't risk creating native extensions build issues anymore 🤷

Reply:
Yep I'm good to get rid of it since it's replicated in CI!

Removed in f727f3b3097d27088ad72cf517f8ea7c6106231c

Reply:
You seem to have forgotten to also remove the lines 69-71 ie when this is called from the `new_release` task

Reply:
Thanks for pointing out the updated lines below – addressed in https://github.com/wordpress-mobile/release-toolkit/commit/d04fe6f73a87d002eca17a2d9d96c9cdb23628bc.